.github/workflows/depviz.yml

```diff
@@ -0,0 +1,15 @@
+on:
+  workflow_dispatch:
+  push:
+
+jobs:
+  generate:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: Doozers/depviz-action@e68e52e8761f640c9dd3bb3dd5859c7f4d29bbe5
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          targets: 'moul/depviz-test'
```

Comment:
Should be a list, not a single string.

Reply:
yes but it doesn't exist, every `with: args` must be string.

Reply:
yes :+1: